<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyEngineClientClass</class>
 <widget class="QMainWindow" name="MyEngineClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyEngineClient</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/material/pic/login/title2.png</normaloff>C:/Users/material/pic/login/title2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="AccountBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>-30</y>
      <width>301</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SignInBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RememberPasswordBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
